src/lib/ChapterList.svelte

```diff
@@ -23,7 +20,7 @@
 </ol>
 {#if startingChapters.length > 2}
   <h1>
-    <Construction {style} />
+    <Icon icon="ic:round-construction" inline />
```

Comment:
@kleinicke Could you or someone from IT take over converting our SVG icons to `@iconify/svelte` as done here and in `src/lib/BasePage.svelte`?

Reply:
I'm not really sure what to do here. Should I figure out a corresponding way to call the same icon with the new framework?